CLR_DEV9_LINUX/PSE.cpp

```diff
@@ -303,8 +304,11 @@ void LoadCoreCLR(string pluginPath, string coreClrFolder)
 
 	if (coreClrFolder.length() == 0)
 	{
-		//coreClrFolder = "/home/air/git/ReadyBin.Release";
-		coreClrFolder = "/home/air/git/ReadyBin.Debug";
+		const char *homedir = getenv("HOME");
+		if ( homedir == NULL ) {
+		    homedir = getpwuid(getuid())->pw_dir;
+		}
+		coreClrFolder = string(homedir) + string("/.config/PCSX2/coreclr");
```

Comment:
CLR_DEV9_CORE.dll is currently found bases on the path passed to `DEV9setSettingsDir` while this assumes it's always in the home directory (it can be configured otherwise) Both should probably be consistent to avoid any confusion if a non-default settings path is encountered.